src/Sulu/Bundle/CategoryBundle/Category/CategoryManager.php

```diff
@@ -314,7 +314,7 @@ public function findChildrenByParentKey($parentKey = null)
     public function save($data, $userId, $locale, $patch = false)
     {
         if ($this->getProperty($data, 'id')) {
-            $categoryEntity = $this->findById($this->getProperty($data, 'id'), $locale);
+            $categoryEntity = $this->findById($this->getProperty($data, 'id'));
```

Comment:
i removed the unnecessary locale-parameter here because it is not part of the method signature